### PR TITLE
Update ErrorSQL.py

### PR DESCRIPTION
**Pull request by gacosta1**
im sorry i had to do this to you Owen. Ever since I told Github to erase the sensitive information my forked acc has been messed up lol

### CODE REVIEW

**Comment by ophilli** [comment]
@gacosta1 1 job man!

**Comment by ophilli** [comment]
@ophilli You're doing great work though - keep it up!

File: Software/scan_key/ErrorSQL.py [hunk 1]
```diff
@@ -7,9 +7,10 @@
 from email.mime.text import MIMEText
 
 def errorSQL(id, errMessage):
+    cnx = pymysql.connect(user='CATS', password='CATS', host='192.168.1.2', database='CATS', autocommit=True)
```
Comment (ophilli) [inline Software/scan_key/ErrorSQL.py:10]: Let's change this back to CATS-SQL.local. I've figured out a way to temporarily resolve the hostname issues.

File: Software/scan_key/ErrorSQL.py [hunk 2]
```diff
@@ -7,9 +7,10 @@
...
 #    cnx = pymysql.connect(user='CATS', password='CATS', host='CATS-SQL.local', database='CATS', autocommit=True)	
-    cnx = pymysql.connect(user='CATS', password='********', host='192.168.0.148', database='CATS', autocommit=True)
-    cursor = cnx.cursor()
+#    cnx = pymysql.connect(user='CATS', password='********', host='192.168.0.148', database='CATS', autocommit=True)
+#    cursor = cnx.cursor()
```
Comment (ophilli) [inline Software/scan_key/ErrorSQL.py]: Looks to me like this should have stayed uncommented.

File: Software/scan_key/ErrorSQL.py (hunk 1 above)
Comment (ophilli) [reply Software/scan_key/ErrorSQL.py:10]: How about we delete this loc and uncomment line 11?

**Review by ophilli — DISMISSED** [review]
Hey G - great work identifying the issue so quickly. Just a few things I'd like to modify.

**Comment by gacosta1** [comment]
I uncommented that one line. You should be able to just merge & pull

**Review by ophilli — APPROVED** [review]

**Pull request merged**